.gitignore

```diff
@@ -1,3 +1,3 @@
-.build/
-vendor/
-composer.lock
+/.build
+/vendor
+/.idea
```

Comment:
I'd prefer to add this to a global configuration - otherwise we end up with a list here for every single IDE.

What do you think?

Reply:
We could do but tbb, i dont think there are that many is it?